### PR TITLE
Add request rate metric to CQL

### PR DESCRIPTION
**Pull request by mpenick**

### CODE REVIEW

**Review by dougwettlaufer — APPROVED** [review]

File: cql/src/main/java/org/apache/cassandra/stargate/metrics/ClientMetrics.java [hunk 1]
```diff
@@ -54,6 +54,7 @@
 
   private AtomicInteger pausedConnections;
   private Meter requestDiscarded;
+  private Meter requestRate;
```
Comment (dimas-b) [inline cql/src/main/java/org/apache/cassandra/stargate/metrics/ClientMetrics.java:57]: nit: this `Meter` produces multiple outputs, so there's no single "rate" in the strict sense I guess. By analogy with `requestDiscarded`, would it be more readable to call the new `Meter` something like `requestsReceived`?

**Review by dimas-b — APPROVED** [review]

File: cql/src/main/java/org/apache/cassandra/stargate/metrics/ClientMetrics.java (hunk 1 above)
Comment (mpenick) [reply cql/src/main/java/org/apache/cassandra/stargate/metrics/ClientMetrics.java:57]: Good suggestions. Thanks. wdyt about `requestsProcessed`? (it's in the current commit)

**Review by dimas-b — APPROVED** [review]

File: cql/src/main/java/org/apache/cassandra/stargate/metrics/ClientMetrics.java (hunk 1 above)
Comment (dimas-b) [reply cql/src/main/java/org/apache/cassandra/stargate/metrics/ClientMetrics.java]: sgtm :+1:

**Pull request merged**